RetailCoder.VBE/UI/CodeExplorer/Commands/AddUserFormCommand.cs

```diff
@@ -57,7 +57,25 @@ private Declaration GetDeclaration(object parameter)
                 node = node.Parent;
             }
 
-            return node == null ? null : ((ICodeExplorerDeclarationViewModel)node).Declaration;
+            return ((ICodeExplorerDeclarationViewModel) node)?.Declaration;
```

Comment:
Perhaps extract some *ModuleMaker* type that the commands would use for this? That way the relatively complex logic below would only be in one place?